### ISSUE TITLE
[CoE Starter Kit - BUG] error when deleting column data in dataverse

### ISSUE THREAD

**Issue by minobrega** [microsoft/coe-starter-kit#8651]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Hello,

Please, I am trying to clear the FLOW ARCHIVE REQUEST IGNORED SINCE column of the Flow table to reconnect the CoE alerts, but when I clear the value of the date field, an error appears on the page and does not let me clear even when editing in Excel when I clear and public. in the dataverse table there is a date 12/31/1988 causing an error in the flows. I can only make changes by putting another date in this column, I cannot delete it. Can you help me clean this column?

![erro CoE coluna ignore](https://github.com/user-attachments/assets/e37cd798-db8b-4559-97bb-0f30719bdcb7)

![erro CoE coluna ignore 2](https://github.com/user-attachments/assets/44937f14-573b-4267-95bc-b0d3e22ceabf)

![erro CoE ignore 2](https://github.com/user-attachments/assets/09ae1196-e36d-4796-b68b-4aef39ea57ab)

### Expected Behavior

I would need to delete the date of some FLOWS in this column, for the maker to receive the charge again, but not allowing it to be deleted causes an error on the page.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.29

### What app or flow are you having the issue with?

Admin | Inactivity notifications v2 (Start Approval for Flows)

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
Hello. I'm not sure why you are seeing that. I am able to clear the data in this field in my managed install. 
You can perhaps write a quick flow to clear them all, you can put null into the field
![image](https://github.com/user-attachments/assets/6fb2e1d7-b859-4958-a46e-236e63316bc6)

**Comment by minobrega** [comment]
Hello,

Jenefer worked!

I had already created a flow to clean, the problem is that I couldn't delete the entire column, there were specific dates that I needed to delete because I had disabled CoE notifications on that data and I couldn't filter because the format of that column is not what I wanted. appears in the dataverse, I put the filter according to what came in Excel and it was!

Thanks a lot for the help!!

![CoE hoje](https://github.com/user-attachments/assets/a8e66700-3a69-44fb-96f4-1d62a7f31cd9)

![CoE hoje 1](https://github.com/user-attachments/assets/dba2de2b-a9c5-4b86-ac72-ac0202073a80)

![CoE hoje 2](https://github.com/user-attachments/assets/a4248633-9b1f-4ade-b61b-65ad70518dee)
